README.gcp.md

```diff
@@ -56,8 +56,8 @@ make run_local_tests
 
 ```
 pip3 install -r requirements.txt
-cd ~/bridge-hackathon/DDS/src
-python3 api.py
+cd ~/bridge-hackathon/DDS/
```

Comment:
The last slash can be dropped

Reply:
It certainly can be! I've noticed that often documentation includes it, presumably to emphasize that it's a directory and not a file. Not necessary for clarification when the command is *cd*, but consistency is a virtue.

Reply:
I've read that `import *` is frowned upon in production code.

Reply:
Commit message updated.